### PR TITLE
[No Ticket][Risk=moderate] Remove billing project buffer feature flag

### PR DESCRIPTION
**Pull request by blrubenstein**

### CODE REVIEW

File: ui/src/app/pages/analysis/notebook-redirect/component.ts [hunk 1]
```diff
@@ -138,15 +137,8 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
...
-      .flatMap(({useBillingProjectBuffer}) => {
-        if (useBillingProjectBuffer) {
-          return this.clusterService.listClusters(this.wsNamespace);
-        }
-        return  userProfileStore.asObservable()
-          .flatMap((profileStore) => {
-            return this.clusterService.listClusters(
-              profileStore.profile.freeTierBillingProjectName);
-          });
+      .flatMap(({}) => {
```
Comment (jmthibault79) [inline ui/src/app/pages/analysis/notebook-redirect/component.ts]: Is there a more appropriate construction to use when the function parameter list is empty, like `.then()` ?

File: ui/src/app/pages/homepage/homepage.tsx [hunk 2]
```diff
@@ -319,16 +315,11 @@ export const Homepage = withUserProfile()(class extends React.Component<
                   <div style={{display: 'flex', flexDirection: 'row', paddingTop: '2rem'}}>
                     <div style={styles.contentWrapperLeft}>
                       <div style={styles.mainHeader}>Researcher Workbench</div>
-                      <TooltipTrigger content={<div>Your Firecloud billing project is still being
```
Comment (jmthibault79) [inline ui/src/app/pages/homepage/homepage.tsx]: Do we have a way to tell when the Billing Buffer is exhausted, to display something similar?

File: ui/src/app/pages/analysis/notebook-redirect/component.ts (hunk 1 above)
Comment (calbach) [reply ui/src/app/pages/analysis/notebook-redirect/component.ts]: serverConfigStore is no longer necessary here:

```
this.loadingSub = this.clusterService.listClusters(...)
  .do((resp) => { ....)
```

File: api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java [hunk 3]
```diff
@@ -167,11 +166,10 @@ public void setUp() {
     // run in the workbench schema only.
     cdrVersion.setCdrDbName("");
 
-    // TODO: Update cluster names to include user IDs once useBillingProjectBuffer is the default.
     String createdDate = Date.fromYearMonthDay(1988, 12, 26).toString();
     testFcCluster =
         new org.pmiops.workbench.notebooks.model.Cluster()
-            .clusterName("all-of-us")
+            .clusterName("all-of-us" + user.getUserId())
```
[the remark this answers is not in the document]
Comment (blrubenstein) [reply api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java]: No! Sorry, I should put the WIP flag on this.

[the remark this answers is not in the document]
Comment (jaycarlton) [reply api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java]: Even better is the [Draft](https://github.blog/2019-02-14-introducing-draft-pull-requests/) status in Github. You can filter them out and everything.

File: ui/src/app/pages/homepage/homepage.tsx (hunk 2 above)
Comment (blrubenstein) [reply ui/src/app/pages/homepage/homepage.tsx]: Not really. I think there's some design going into what to do there.

Comment (jaycarlton) [reply ui/src/app/pages/homepage/homepage.tsx]: Please add a story to capture that design and/or implementation activity if there's not one already.

Comment (blrubenstein) [reply ui/src/app/pages/homepage/homepage.tsx]: https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=30&view=planning.nodetail&selectedIssue=RW-3424&epics=visible&selectedEpic=RW-1205

**Pull request merged**